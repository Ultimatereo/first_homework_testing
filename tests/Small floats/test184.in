3
4.2e-07 1.63e-07 2.14e-07 2.72e-07
3.98e-07 3.94e-07 1.8e-08 1.2e-07
2.4e-07 2.02e-07 3.32e-07 4.89e-07
