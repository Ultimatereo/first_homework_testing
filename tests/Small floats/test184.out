-0.220379
0.465558
1.34894
